src/components/DefaultImage.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react';
+
+type DefaultImageProps = {
+  /**
+   * The address of the image you want to display.
+   */
+  imageSrc: string;
+  /**
+   * The address of a default image to display if there's an issue displaying the prefered image address.
+   */
+  defaultSrc?: string | null;
+};
+
+type DefaultImageDefaultProps = {
+  defaultSrc: string | null;
+}
+
+type DefaultImageState = {
+  src: string | null;
+};
+
+/**
```

Comment:
I would include some of what the original component's comment says about the img properties, etc. The original DisappearingImage component's comment is here:
`Component that will hide an image if it can't be displayed due to, e.g.,
a URL to a non-existant file. You can pass in any props that work for
a standard <img> tag and they'll be added to the inserted image.`